lib/pure/net.nim

```diff
@@ -44,46 +44,44 @@
 ## After you create a socket with the `newSocket` procedure, you can easily
 ## connect it to a server running at a known hostname (or IP address) and port.
 ## To do so over TCP, use the example below.
-##
-## .. code-block:: Nim
-##   var socket = newSocket()
-##   socket.connect("google.com", Port(80))
-##
-## For SSL, use the following example (and make sure to compile with `-d:ssl`):
-##
-## .. code-block:: Nim
-##   var socket = newSocket()
-##   var ctx = newContext()
-##   wrapSocket(ctx, socket)
-##   socket.connect("google.com", Port(443))
-##
+
+runnableExamples("-r:off"):
+  var socket = newSocket()
```

Comment:
```suggestion
  let socket = newSocket()
```
`Socket` is a `ref object`, so `var` is not necessary. Ditto for all the other cases.

Reply:
i know, it was pre-existing though; anyway, i agree that codeblock=>runnableExamples PRs should also improve underlying examples, so I've fixed this.